apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/completionWorks.js

```diff
@@ -99,6 +102,7 @@ const generateTicketDocumentOfCompletionWorks = async ({ task, baseAttrs, contex
         },
         unit: {
             name: get(ticket, 'unitName') || '',
+            type: (unitType ? i18n(`pages.condo.ticket.field.unitType.act.${unitType}`, { locale }) : '').toLowerCase(),
```

Comment:
This translation does not apply to ticket page.
It would бe better to use a key like this: `field.unitType.{type}.genitive` or come up with something related to the ongoing process